setup.py

```diff
@@ -11,7 +11,15 @@
 with open('HISTORY.rst') as history_file:
     history = history_file.read()
 
-requirements = ['Click>=6.0', 'toolz', 'grpcio', 'googleapis-common-protos']
+requirements = [
```

Comment:
i don't like that we have 3 places with deps `setup.py`, `requirements.txt`, and `requirements_dev.txt`. `requirements` in `setup.py` should be generated from `requirements.txt`

Reply:
somethink like should be ok for now
```
with open('requirements.txt') as f:
    requirements = f.read().splitlines()
```

Reply:
done